FileManipulator.Test/GlobalJsonManipulatorTest.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Messerli.FileManipulatorAbstractions;
+using Messerli.FileSystem;
+using Messerli.Test.Utility;
+using Xunit;
+
+namespace Messerli.FileManipulator.Test
+{
+    public sealed class GlobalJsonManipulatorTest
+    {
+        private const string FileName = "global.json";
+        private const string MsBuildSdkNugetPackageId = "Microsoft.Build.NoTargets";
+        private const string MsBuildSdkVersion = "1.0.0";
+        private static readonly string NewLine = Environment.NewLine;
+
+        private static readonly GlobalJsonModification AddMsBuildSdkModification =
```

Comment:
This is a static modification, not a method. Why is it a verb?

Reply:
"Add" is not intended as a verb here

Reply:
https://github.com/messerli-informatik-ag/meta-generator/pull/14/commits/5a1213aa05724d351632a9cc108c4581b9b3d180